src/app/models/propTypes/chartbeatAnalytics/index.js

```diff
@@ -0,0 +1,23 @@
+import { string, shape, number, bool, oneOf, oneOfType } from 'prop-types';
+
+const baseChartbeatPropTypes = {
+  domain: string.isRequired,
+  sections: string.isRequired,
+  uid: number.isRequired,
+  title: string.isRequired,
+  virtualReferrer: oneOfType([string, oneOf([null])]),
+  idSync: shape({
+    bbc_hid: string,
+  }),
+};
+
+export const canonicalChartbeatPropTypes = {
```

Comment:
Ideally these would have `shape` around them, to save having to use it in the files at use it